src/Persistence/Yaml/NamedObjectEmitter.cs

```diff
@@ -14,38 +14,54 @@ namespace Microsoft.PowerPlatform.PowerApps.Persistence.Yaml;
 ///    Property1: Value1
 ///    Property2: Value2
 /// </summary>
-/// <typeparam name="T"></typeparam>
-internal class NamedObjectEmitter<T> : ChainedEventEmitter
-    where T : class
+internal class NamedObjectEmitter : ChainedEventEmitter
 {
-    private readonly Func<T, string?> _nodeNameProvider;
+    private Stack<bool> _isNamedObject = new Stack<bool>();
```

Comment:
what's the lifetime of `NamedObjectEmitter` ? What about multithreading?

Reply:
This emitter is not actually used at all. I kept it for future reference